modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -690,6 +863,21 @@ private void _validateInvokerBundle(String message, boolean system)
 	private static final Log _log = LogFactoryUtil.getLog(
 		ObjectValidationRuleLocalServiceImpl.class);
 
+	private static final List<String> _allowedObjectFieldBusinessTypes =
```

Comment:
@mateussgn can we rename it to something like: `uniqueCompositeKeyObjectFieldBusinessTypes`?